### PR TITLE
Add OperationTypeManageSellOffer operation type to history_assets export

### PR DESCRIPTION
**Pull request by cayod**
With the necessity of getting all the assets in history_assets table we add `xdr.OperationTypeManageSellOffer` operation type to export assets, basically we will get all the operations that has `OperationTypeManageSellOffer` or `OperationTypePaymentoperation` type and use in the transform.

### CODE REVIEW

File: internal/transform/asset.go [hunk 1]
```diff
@@ -15,16 +15,29 @@ func TransformAsset(operation xdr.Operation, operationIndex int32, transactionIn
...
 	opType := operation.Body.Type
-	if opType != xdr.OperationTypePayment {
+	if opType != xdr.OperationTypePayment && opType != xdr.OperationTypeManageSellOffer {
 		return AssetOutput{}, fmt.Errorf("operation of type %d cannot issue an asset (id %d)", opType, operationID)
 	}
 
-	op, ok := operation.Body.GetPaymentOp()
-	if !ok {
-		return AssetOutput{}, fmt.Errorf("could not access Payment info for this operation (id %d)", operationID)
+	op := xdr.Asset{}
```
Comment (sydneynotthecity) [inline internal/transform/asset.go:22]: this variable name is a little confusing. Would prefer if it were renamed something like `asset` instead of `op`

**Review by sydneynotthecity — APPROVED** [review]
Looks good

**Comment by sydneynotthecity** [comment]
@cayod have you had the chance to review my comments? Let me know when we're ready to merge

**Comment by cayod** [comment]
> @cayod have you had the chance to review my comments? Let me know when we're ready to merge

It's Ready for merge.

**Pull request merged**